examples/microprofile/hello-world-explicit/src/main/java/io/helidon/microprofile/example/helloworld/explicit/Main.java

```diff
@@ -34,7 +34,6 @@ private Main() {
      */
     public static void main(String[] args) {
         Server server = Server.builder()
-                .addApplication(HelloWorldApplication.class)
                 // using a customized helidon config instance (in this case the default...)
```

Comment:
The message is wrong - this uses the MP Config, not Helidon config

Reply:
Fixed